src/main/java/frc/robot/commands/lowerElevator.java

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.elevator;
+import frc.robot.subsystems.subWristem;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class lowerElevator extends InstantCommand {
+  elevator Elevator = new elevator();
+  subWristem intake = new subWristem();
+  public lowerElevator() {
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
```

Comment:
Subsystems should be passed in as parameters and the addRequirments method should be used